### DISCUSSION TITLE
手搓一个简历模板吧！React 实践踩坑记录 | Longlin's Blog

### DISCUSSION THREAD

**Discussion by giscus** [longlin10086/blog#10] [Blog comments]
# 手搓一个简历模板吧！React 实践踩坑记录 | Longlin's Blog

resume-web 开发过程踩坑

https://blog.longlin.tech/posts/resume-develop/

**Comment by ghost** [comment]
好看(≧∀≦)ゞ

![pic](https://tgs.sgp1.digitaloceanspaces.com/watora_potofu/star_struck_c768c103.gif)

**Reply by longlin10086** [reply]
谢谢！🥰
